### PR TITLE
Update Android Scenario Startup Flow

### PR DESCRIPTION
**Pull request by LoopedBard3**
Update the Android Scenario Startup Flow to use adb instead of test runners for more accurate startup times and the ability to test start up to first drawn frame on any apk. This effort is part of #2075 as it more easily enables the ability for Maui app testing.

### CODE REVIEW

File: src/scenarios/shared/runner.py [hunk 1]
```diff
@@ -328,48 +338,129 @@ def run(self):
...
+                'am',
+                'start-activity',
+                '-W',
+                '-n',
+                activityname
+            ]
+            testRun = RunCommand(startAppCmd, verbose=True)
+            testRun.run()
+            testRunStats = re.findall(runSplitRegex, testRun.stdout) 
+            getLogger().info(f"Test run activity: {testRunStats[3]}")
```
Comment (DrewScoggins) [inline src/scenarios/shared/runner.py:397]: What is testRunStats[3]? Can you break down what the different groups of this regex are.

Comment (LoopedBard3) [reply src/scenarios/shared/runner.py:408]: Added a break down the data split at the top of this switch section (line 310)

**Review by DrewScoggins — APPROVED** [review]
LGTM

**Comment by LoopedBard3** [comment]
Errors are unrelated, merging

**Pull request merged**